skeletons/src/dsAudio.c

```diff
@@ -565,6 +565,13 @@ dsError_t dsAudioFormatUpdateRegisterCB(dsAudioFormatUpdateCB_t cbFun)
   return (dsError_t)0;
```

Comment:
please remove the stub implementation from `skeletons/src/dsAudio.c` which is removed from the header

Reply:
done